### PR TITLE
add creator for robot

### PR DESCRIPTION
**Pull request by wy65701436**
add the creator for robot creation

Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Review by chlins — APPROVED** [review]
lgtm

File: api/v2.0/swagger.yaml [hunk 1]
```diff
@@ -7846,6 +7846,9 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/RobotPermission'
+      Creator:
```
Comment (stonezdj) [inline api/v2.0/swagger.yaml:7849]: creator

File: src/server/v2.0/handler/robot.go [hunk 2]
```diff
@@ -62,12 +62,18 @@ func (rAPI *robotAPI) CreateRobot(ctx context.Context, params operation.CreateRo
 		return rAPI.SendError(ctx, err)
 	}
 
+	secCtx, err := rAPI.GetSecurityContext(ctx)
```
Comment (stonezdj) [inline src/server/v2.0/handler/robot.go:65]: It is better to add creator it in the robotManager level

**Review by stonezdj — APPROVED** [review]

File: src/controller/robot/controller_test.go [hunk 3]
```diff
@@ -2,6 +2,8 @@ package robot
 
 import (
 	"context"
+	"github.com/goharbor/harbor/src/common/security"
```
Comment (zyyw) [inline src/controller/robot/controller_test.go:5]: please format this import

**Review by zyyw — APPROVED** [review]

**Pull request merged**